### PR TITLE
Add remapColor, AS Disney Glass shaders

### PR DESCRIPTION
**Pull request by luisbarrancos**
The remapColor node depends on a custom (ramp) exporter, added for review with 
https://github.com/appleseedhq/appleseed-maya/pull/21

### CODE REVIEW

File: src/appleseed.shaders/CMakeLists.txt [hunk 1]
```diff
@@ -159,6 +159,9 @@ set (appleseed_maya_osl_sources
      src/maya/as_maya_addMatrix.osl
      src/maya/as_maya_anisotropic.osl
      src/maya/as_maya_areaLight.osl
+     src/maya/as_maya_asCarPaint.osl
```
Comment (est77) [inline src/appleseed.shaders/CMakeLists.txt]: You need to remove this one

File: src/appleseed.shaders/src/maya/as_maya_asDisneyMaterial.osl [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+        string help = "Vector tangent field map, with XY in R,G channels"
+    ]],
+    float in_sheenAmount = 0.0
+    [[
+        float min = 0.0,
+        float max = 10.0,
+        float softmin = 0.0,
+        float softmax = 1.0,
+        float slidermin = 0.0,
+        float slidermax = 1.0,
```
Comment (est77) [inline src/appleseed.shaders/src/maya/as_maya_asDisneyMaterial.osl]: maybe this should be 10?

File: src/appleseed.shaders/src/maya/as_maya_asGlass.osl [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    [[
+        string maya_attribute_name = "color",
+        string maya_attribute_short_name = "c",
+        string label = "Transmittance Color",
+        string page = "Surface Transmittance Parameters"
+    ]],
+    float in_transmittanceAmount = 0.8
+    [[
+        float min = 0.0,
+        float max = 100.0,
```
Comment (est77) [inline src/appleseed.shaders/src/maya/as_maya_asGlass.osl]: We should limit this to 1, I think.

File: src/appleseed.shaders/src/maya/as_maya_asGlass.osl [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+                    tangent,
+                    in_anisotropyAngle * M_2PI,
+                    point(0),
+                    point(in_normalCamera));
+            }
+        }
+
+        string distribution = (in_distribution) ? "ggx" : "beckmann";
+
+        out_outColor = in_transmittanceAmount *
```
Comment (est77) [inline src/appleseed.shaders/src/maya/as_maya_asGlass.osl]: in_transmittanceAmount should go inside the closure, multiplied with in_surfaceTransmittance.
In the BSDF there's a special treatment for surfacet transmittance.

File: src/appleseed.shaders/CMakeLists.txt (hunk 1 above)
Comment (luisbarrancos) [inline src/appleseed.shaders/CMakeLists.txt]: @est77 my bad, i forgot that one

File: src/appleseed.shaders/src/maya/as_maya_asGlass.osl (hunk 4 above)
Comment (luisbarrancos) [inline src/appleseed.shaders/src/maya/as_maya_asGlass.osl]: @est77 ok. The 100.0 upper bound as a typo as well, should've been 1.0.

**Pull request merged**